src/neo/ProtocolSettings.cs

```diff
@@ -10,8 +10,8 @@ public class ProtocolSettings
         public uint Magic { get; }
         public byte AddressVersion { get; }
         public string[] StandbyCommittee { get; }
-        public byte MaxCommitteeMembersCount { get; }
```

Comment:
Who currently can edit these values `MaxCommitteeMembersCount` and `MaxValidatorsCount`?
They are set by default, right, on the protocol?
Perhaps we need also the Committee to vote for these values onchain, at least.

Reply:
No one can edit these values. They are hard-coded.